docs/v2/api-endpoints/get_address_details.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Get the details of a b58 encoded public address
```

Comment:
None of this particular info is sensitive, since it's just decoding unencrypted hex encoded b58 protobuf data. But yeah, maybe a sensitivity warning on the more important things would be good